src/Message/Content/Image.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Message\Content;
+
+final readonly class Image implements ContentInterface
+{
+    public function __construct(public string $image)
```

Comment:
I would rename it to 
```suggestion
    /** @param $url An URL like http..... or a data url.... like ....
    public function __construct(public string $url)
```
WDYT?

Reply:
Had chosen the name because data url and http url are valid. But yeah. You are right. Sounds a bit better, at least together with a comment.